subsys/emds/Kconfig

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig EMDS
+	bool "Emergency Data Storage [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	depends on PARTITION_MANAGER_ENABLED
+	depends on FLASH_MAP
+	default n
+	help
+	  Enable Emergency Data Storage
+
+if EMDS
+
+config EMDS_SECTOR_COUNT
+	int "Sector count of the emergency data storage area"
+	default 1
+	help
+	  Number of sectors used for the emergency data storage area
+
+config EMDS_THREAD_STACK_SIZE
+	int "Stack size for the emergency data storage thread"
+	default 500
+	help
+	  Size of the stack that is initialized by the emergency data storage
+
+config EMDS_THREAD_PRIORITY
+	int "Priority for the emergency data storage thread"
+	default 0
+	help
+	  Cooperative priority for the emergency data storage thread. This will
+	  be used through K_PRIO_COOP(x), that means higher value gives lower
+	  priority.
+
+config EMDS_FLASH_TIME_WRITE_ONE_WORD_US
+	int "Time to write one word into flash"
+	default 41
+	help
+	  Max time to write one word (4 bytes) in flash (in microseconds)
+
+config EMDS_FLASH_TIME_ENTRY_OVERHEAD_US
+	int "Time to schedule write of one entry"
+	default 300
+	help
+	   Max time to prepare the write of each entry (in microseconds)
+
+config EMDS_FLASH_TIME_BASE_OVERHEAD_US
```

Comment:
I think this should not be a config option but instead should be defined in the emds.h. Configuring wrong values here leads to the failure of EMDS functionality.

Reply:
I disagree that this should be in the emds.h file. That means the user has to change the code in the library dependent on the chip the user is using. That belongs to configuration of the users project.

Reply:
What I am saying, these values should be calculated automatically in the EMDS header. These values do not change for a given chip. Users should not have to enter them manually.